### PR TITLE
Update zmq to 4.3.4

### PR DESCRIPTION
**Pull request by gruve-p**
Addresses https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-13132 and https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-15166

### CODE REVIEW

File: depends/packages/zeromq.mk [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package=zeromq
-$(package)_version=4.3.1
+$(package)_version=4.3.4
 $(package)_download_path=https://github.com/zeromq/libzmq/releases/download/v$($(package)_version)/
 $(package)_file_name=$(package)-$($(package)_version).tar.gz
-$(package)_sha256_hash=bcbabe1e2c7d0eec4ed612e10b94b112dd5f06fcefa994a0c79a45d835cd21eb
+$(package)_sha256_hash=c593001a89f5a85dd2ddf564805deb860e02471171b3f204944857336295c3e5
```
Comment (fanquake) [inline depends/packages/zeromq.mk:5]: The trailing whitespace here will not only trip up the CI linter, but the package will fail to download at all. How was this tested?

**Review by fanquake — COMMENTED** [review]
Do these effect us? We been building with CURVE authentication disabled [since we updated ZeroMQ in depends to 4.1.5](https://github.com/bitcoin/bitcoin/commit/a615386552f9981dc024cc4a0df758f8b0fda4d8).

File: depends/packages/zeromq.mk (hunk 1 above)
Comment (gruve-p) [reply depends/packages/zeromq.mk:5]: This was tested locally on another machine, will remove the trailing whitespace

**Comment by MarcoFalke** [comment]
Please squash your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by hebasto** [comment]
> Do these effect us? We been building with CURVE authentication disabled [since we updated ZeroMQ in depends to 4.1.5](https://github.com/bitcoin/bitcoin/commit/a615386552f9981dc024cc4a0df758f8b0fda4d8).

From the CVE descriptions I see no point in the `libzmq` update.

**Comment by MarcoFalke** [comment]
Are there other fixes that make the update worthwhile?

**Comment by gruve-p** [comment]
Feel free to close this PR if you see no point in the update

**Comment by laanwj** [comment]
I don't think it makes sense to track minor relases here, if the changes don't affect our usage.

**Pull request closed without merging**